config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root 'users#index'
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show, :new, :create] do
```

Comment:
- [ ] Based on the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#make-sure-that-when-you-run-the-rails-routes-command-in-your-console-you-can-see-the-following-output), please make sure only the index and show routes for posts are exposed 👍🏿

Reply:
Thank you so much